### DISCUSSION TITLE
MDN 翻訳に関するポリシーみたいなものはありますか?

### DISCUSSION THREAD

**Discussion by hiroshi** [mozilla-japan/translation#636] [Q&A] [answered]
具体的には
https://developer.mozilla.org/ja/docs/Glossary/Synthetic_monitoring
の日本語訳「合成モニタリング」という訳語に違和感があります。

「合成」の意味を辞書で調べるといずれの意味も「２つ以上の...」という説明があります。
しかし、この synthetic は「２つ以上の何かをあわせる」的な意味は無く「人工」とか「人為的な」という意味だと思います。
実験的、作為的に行った事象を観測するということ。
real user monitoring の逆ということ。

ネットで検索するとたくさんの「合成モニタリング」 または「合成監視」(これも監視より観測のほうが意味がわかる)という言葉がヒットするので意味的に分かりにくくても、すでにそういう訳がたくさん使われているので倣ったのかとは思われます。

そこで質問です。 MDN の日本語訳のポリシーとしてはどちらでしょうか?
- A) わかりにく訳語でもすでに広まってるものに倣う
- B) もっとわかりやすい訳語に置き換える

もちろん、ケースバイケースということになるのかもしれません。

- B) をとり得る可能性があるのでしょうか?
- より具体的には、私が「実験的観測」(これがベストだとは思いませんが synthetic monitoring の意味を理解すれば「合成モニタリング」よりはなるほどと思うと考えます。)に置き換えた pull request を作成した場合に取り込まれる可能性はあるのでしょうか?

以上気になってしまったので質問させていただきました。 
ややこしいことを言ってすみません。とくに組織的なポリシーが無いのでしたら、個人的にはこう思うみたいなコメント歓迎です。

翻訳むずかしいですよね。プロの翻訳家が翻訳した海外の小説を読んでも意味がわからないところがあったりして、調べると誤訳みたいなのはよくありますしね。

**Comment by marsf** [comment, marked as answer]
> MDN の日本語訳のポリシーとしてはどちらでしょうか?
>  A) わかりにく訳語でもすでに広まってるものに倣う
>  B) もっとわかりやすい訳語に置き換える

「MDN の」訳語のポリシーというものは特にありません。
わかりやすい訳語にするのは当然のこととして、A) のようにすでに広まっていて、その表現でどのようなものかが理解されているのであれば、一般的に使われている訳語を使うこともありだと思います。
ポリシーではなく、翻訳者が訳語を選ぶのに迷ったときに、ググってヒット数の多いほうを選ぶ (慣習に従う) ことがあります。

今回の「[Synthetic monitoring](https://en.wikipedia.org/wiki/Synthetic_monitoring)」は、ユーザーの行動と経路をシミュレートして繰り返し実施し、サイトのパフォーマンス (動作が遅くなったりダウンしたりしないか) を監視することだそうです。

Synthetic についてジーニアス英和辞典では、
> synthetic ・・・ 1. 統合の、統合的な (⇔ Analytic)　2. 〈化学用語で〉合成の 3. 本物でない、つくりものの (⇔ genuine)

コウビルド英英辞典では、
> Synthetic  products are made from chemicals or artificial sustances rather than from natural ones.

Wikipedia には、
> This type of monitoring does not require actual traffic, thus the name synthetic.

とあります。
ご指摘の通り、実際のユーザーの行動ではないもの、作為的なデータによりパフォーマンスを監視するものらしいです。音声合成の「合成」と同じ意味合いではないかと思います。

というわけで、技術的に適切な表現の訳語であれば B) のように置き換えるのもありだと思いますが、数年でもビジネスが成り立っているので、新しい訳語がマーケットに受け入れられるかどうかは微妙だと思います。

**Reply by hiroshi** [reply]
> 音声合成の「合成」と同じ意味合いではないかと思います。

なるほど、"speech synthesis" は「音声合成」で浸透している感じしますね。複数の音声の波形が合成されているという意味でなく機械により生成された音声という意味で。人造ゴムである合成ゴムも同様。

個人的にはスッキリしませんが、主観の問題でしょうね。

いずれにせよ返信頂けて、大変ありがとうございました!!
